pkg/pillar/scripts/device-steps.sh

```diff
@@ -491,10 +494,15 @@ fi
 
 echo "$(date -Ins -u) Done starting EVE version: $(cat /run/eve-release)"
 
-# If there is a USB stick inserted and debug.enable.usb is set, we periodically
-# check for any usb.json with DevicePortConfig, deposit our identity,
-# and dump any diag information
+# XXX Debug record goroutines
 while true; do
-    access_usb
-    sleep 300
+    pkill -USR1 zedbox
+    sleep 30
+    total=$(grep -c goroutine /persist/agentdebug/zedbox/sigusr1)
+    grpc=$(grep -c 'created by google.golang.org/grpc.newClientStream' /persist/agentdebug/zedbox/sigusr1)
+    pid=$(cat /run/zedbox.pid)
+    rss=$(grep -i rss /proc/"$pid"/status)
+    echo "$(date -Ins -u) total goroutines $total grpc.newClientStream $grpc $rss" >>/persist/log/goroutines
```

Comment:
This file might grow big over a period of time.

Reply:
This code didn't belong in the PR. Thanks for catching it. Updated the PR.